CONTRIBUTING.md

```diff
@@ -1,12 +1,24 @@
-## Developer setup instructions
+## Developer instructions
 
-1. `git clone git@github.com:web-animations/web-animations-next.git`
+### Setup
+
+1. Fork web-animations/web-animations-js.
+1. `git clone git@github.com:$GITHUB_USER/web-animations-js.git`
 1. `git submodule update --init --recursive` (Necessary for running tests.)
 1. Install [node](https://nodejs.org/en/) and make sure `npm` is in your $PATH
 1. Run `npm install` in the respository to pull in development dependencies.
 1. Run `npm install -g grunt grunt-cli` to get the build tools for the command line.
+
+### Contributing
+
+Note that development should occur against the `dev` branch, not `master`. This
+is the default target for pull requests.
+
+1. In your fork of web-animations-js, `git checkout dev` or create a new branch whose parent is dev.
 1. Run `grunt` to build the polyfill.
 1. Run `grunt test` to run polyfill and web-platform-tests tests.
+1. Commit changes to your fork.
+1. Create a pull request from your fork of web-animations-js to web-animations/web-animations-js/dev.
```

Comment:
Maybe link to https://github.com/web-animations/web-animations-js/tree/dev for clarity.